.openzeppelin/addresses-localhost.json

```diff
@@ -0,0 +1 @@
"PayMerchantHandler":{"proxy":"0x84eA74d481Ee0A5332c457a4d796187F6Ba67fEB","contractName":"PayMerchantHandler"},"RegisterMerchantHandler":{"proxy":"0xa82fF9aFd8f496c3d6ac40E2a0F282E47488CFc9","contractName":"RegisterMerchantHandler"},"SplitPrepaidCardHandler":{"proxy":"0x851356ae760d987E095750cCeb3bC6014560891C","contractName":"SplitPrepaidCardHandler"},"TransferPrepaidCardHandler":{"proxy":"0x95401dc811bb5740090279Ba06cfA8fcF6113778","contractName":"TransferPrepaidCardHandler"},"SetPrepaidCardInventoryHandler":{"proxy":"0x70e0bA845a1A0F2DA3359C97E0285013525FFC49","contractName":"SetPrepaidCardInventoryHandler"},"RemovePrepaidCardInventoryHandler":{"proxy":"0x99bbA657f2BbC93c02D617f8bA121cB8Fc104Acf","contractName":"RemovePrepaidCardInventoryHandler"},"SetPrepaidCardAskHandler":{"proxy":"0x8f86403A4DE0BB5791fa46B8e795C547942fE4Cf","contractName":"SetPrepaidCardAskHandler"},"AddPrepaidCardSKUHandler":{"proxy":"0x5eb3Bc0a489C5A8288765d2336659EbCA68FCd00","contractName":"AddPrepaidCardSKUHandler"},"BridgeUtils":{"proxy":"0x809d550fca64d94Bd9F66E60752A544199cfAC3D","contractName":"BridgeUtils"},"TokenManager":{"proxy":"0x1291Be112d480055DaFd8a610b7d1e203891C274","contractName":"TokenManager"},"MerchantManager":{"proxy":"0xb7278A61aa25c888815aFC32Ad3cC52fF24fE575","contractName":"MerchantManager"},"SupplierManager":{"proxy":"0x82e01223d51Eb87e16A03E24687EDF0F294da6f1","contractName":"SupplierManager"},"SPEND":{"proxy":"0x7969c5eD335650692Bc04293B07F5BF2e7A673C0","contractName":"SPEND"},"DAIOracle":{"proxy":"0xc351628EB244ec633d5f21fBD6621e1a683B1181","contractName":"ChainlinkFeedAdapter"},"CARDOracle":{"proxy":"0xFD471836031dc5108809D173A067e8486B9047A3","contractName":"ChainlinkFeedAdapter"},"RewardManager":{"proxy":"0x1429859428C0aBc9C2C47C8Ee9FBaf82cFA0F20f","contractName":"RewardManager"},"RewardSafeDelegateImplementation":{"proxy":"0xB0D4afd8879eD9F52b28595d31B441D079B2Ca07","contractName":"RewardSafeDelegateImplementation"},"AddRewardRuleHandler":{"proxy":"0x922D6956C99E12DFeB3224DEA977D0939758A1Fe","contractName":"AddRewardRuleHandler"},"LockRewardProgramHandler":{"proxy":"0x1fA02b2d6A771842690194Cf62D91bdd92BfE28d","contractName":"LockRewardProgramHandler"},"RegisterRewardProgramHandler":{"proxy":"0x04C89607413713Ec9775E14b954286519d836FEf","contractName":"RegisterRewardProgramHandler"},"RegisterRewardeeHandler":{"proxy":"0x21dF544947ba3E8b3c32561399E88B52Dc8b2823","contractName":"RegisterRewardeeHandler"},"UpdateRewardProgramAdminHandler":{"proxy":"0xD8a5a9b31c3C0232E196d518E89Fd8bF83AcAd43","contractName":"UpdateRewardProgramAdminHandler"},"PayRewardTokensHandler":{"proxy":"0x51A1ceB83B83F1985a81C295d1fF28Afef186E02","contractName":"PayRewardTokensHandler"},"DAIUSDFeed":{"proxy":"0x8198f5d8F8CfFE8f9C413d98a0A55aEB8ab9FbB7","contractName":"ManualFeed"},"ETHUSDFeed":{"proxy":"0x0355B7B8cb128fA5692729Ab3AAa199C1753f726","contractName":"ManualFeed"},"CARDUSDFeed":{"proxy":"0x202CCe504e04bEd6fC0521238dDf04Bc9E8E15aB","contractName":"ManualFeed"}}
```

Comment:
@FadhlanR would it be possible to pretty print this JSON file like we do for the other address-*.json files? it makes it a littler easier to introspect that way